docs/cri/config.md

```diff
@@ -247,7 +247,9 @@ version = 3
             ShimCgroup = ''
 
     [plugins.'io.containerd.cri.v1.runtime'.cni]
-      bin_dir = '/opt/cni/bin'
+      # DEPRECATED, use `bin_dirs` instead.
```

Comment:
```suggestion
      # DEPRECATED, use `bin_dirs` instead (since containerd v2.1).
```

https://github.com/containerd/containerd/blob/main/RELEASES.md#deprecated-features needs to be updated too

Reply:
updated